models/__init__.py

```diff
@@ -137,7 +137,7 @@ class Association(DarwinModel):
     associated_service  = ForeignKeyField(AssociationService, related_name="associated_service_set")
     tiploc              = CharField()
     category            = CharField(choices=ASSOCIATION_TYPE)
-    cancelled           = BooleanField()
+    cancelled           = BooleanField(null=True, default=False)
```

Comment:
I might be wrong but I think default doesn't make it null able by default, or at least I had this error last time.